lib/govuk_design_system_formbuilder/builder.rb

```diff
@@ -586,8 +586,8 @@ def govuk_check_box(attribute_name, value, hint_text: nil, label: {}, link_error
     #   = f.govuk_submit "Proceed", prevent_double_click: true do
     #     = link_to 'Cancel', some_other_path, class: 'govuk-button__secondary'
     #
-    def govuk_submit(text = config.default_submit_button_text, warning: false, secondary: false, prevent_double_click: true, validate: false, &block)
-      Elements::Submit.new(self, text, warning: warning, secondary: secondary, prevent_double_click: prevent_double_click, validate: validate, &block).html
+    def govuk_submit(text = config.default_submit_button_text, warning: false, secondary: false, custom_class: nil, prevent_double_click: true, validate: false, &block)
+      Elements::Submit.new(self, text, warning: warning, secondary: secondary, custom_class: custom_class, prevent_double_click: prevent_double_click, validate: validate, &block).html
```

Comment:
[Some other helpers](https://github.com/DFE-Digital/govuk_design_system_formbuilder/blob/master/lib/govuk_design_system_formbuilder/builder.rb#L392) in the gem already contain this functionality via the `classes` param. I think this should be called `classes` too for consistency.